### PR TITLE
Automatically notify users of new mail

### PR DESCRIPTION
**Pull request by salto-edit**
LGTM :dollar:

### CODE REVIEW

**Comment by salto-edit** [comment]
LGTM

File: bot.py [hunk 1]
```diff
@@ -31,6 +31,13 @@
 config = {}
 
 
+def privmsg(receiver, text):
+    '''RFC 1459 PRIVMSG private messages
```
Comment (ltworf) [inline bot.py:35]: This function documentation isn't very documentative

Comment (salto-edit) [inline bot.py:35]: I don't think you can be more precise than that. What do you mean by "documentative" then?

Comment (ltworf) [inline bot.py:35]: Saying what the function does, and what the parameters do.

**Comment by ltworf** [comment]
Oh you are improving, this one didn't need to be fixed after the merge! Good!

**Pull request merged**